sale_order_weight/reports/report_sale_order.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+     Copyright 2016 Andrea Cometa - Apulia Software
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/gpl.html).
+ -->
+<odoo>
+
+    <template
+        id="report_saleorder_document"
+        inherit_id="sale.report_saleorder_document"
+    >
+
+        <xpath expr="//t[@t-set='address']" position="after">
+            <t
+                t-set="weight_uom_name"
+                t-value="doc.env['product.template']._get_weight_uom_name_from_ir_config_parameter()"
```

Comment:
I would delegate these lookups to a method on sale_order `_get_report_weight_params` or smth similar.
Eg: `weight_uom_name, weight_precision = doc._get_report_weight_params()`

Reply:
Hi. 

It's a copy past of the existing V14 module https://github.com/OCA/sale-reporting/blob/14.0/sale_order_weight/views/sale_report.xml#L15-L22
I don't see the interest to define a new function in python code. I see it more verbose. Could you elaborate ? 
Thanks !

Reply:
Regarding "verbosity" I'd say the contrary :wink: 
Regarding "consistency" w/ those examples... fine.